### ISSUE TITLE
ViaFabric 1.14.4 on 20w18a server kick on blockplacement

### ISSUE THREAD

**Issue by RaphiMC** [ViaVersion/ViaFabric#18]
**Describe the bug, provide any errors**
Setup: ViaFabric 1.14.4 latest 1.14.4 version and latest VV VB and VR builds
If I join a 20w18a Server with ViaFabric and take a block and place it in the air the server kicks the player.

**How can we reproduce it?**
1. Select version 1.16.
2. Join a 20w18a server.
3. Take a block and place it in the air.
4. The server will kick you.

**Expected behaviour**
Nothing should happen. No kick.

**Screenshots**
https://streamable.com/vwyz17

**ViaVersion Dump:**
- Can't do that in ViaFabric ([ViaFabric-Config](https://pastebin.com/Qb4rVZHX))

**Additional server info**
No plugins were present on the server.

**Comment by creeper123123321** [comment]
Can this bug be reproduced on 1.14.4 Spigot server?

**Comment by RaphiMC** [comment]
Do you mean joining with ViaFabric 1.14.4 on an Spigot 1.14.4 Server?

**Comment by creeper123123321** [comment]
No, joining with 20w18a vanilla client on Spigot 1.14.4 server with
ViaVersion

**Comment by RaphiMC** [comment]
The same bug doesn't occur on a spigot 1.14.4 server with an 20w18a client.
